core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraIdentityProvider.java

```diff
@@ -175,7 +177,8 @@ public void createClient(
                         "--response-types", "token,code,id_token",
                         "--scope", "openid,offline",
                         "--token-endpoint-auth-method", tokenEndpointAuthMethod.getValue(),
-                        "--callbacks", callbackUrl)
+                        "--callbacks", callbackUrl,
+                        "--post-logout-callbacks", logoutCallbackUrl)
```

Comment:
Probably we need to update docs to add information about registering post-logout url